gtk/Cargo.toml

```diff
@@ -64,6 +64,7 @@ glib = { path = "../glib" }
 gdk = { path = "../gdk" }
 gdk-pixbuf = { path = "../gdk-pixbuf" }
 pango = { path = "../pango" }
+paste = "1.0"
```

Comment:
Add a hidden re-export of `paste` to glib and use that in the macro. It's not very nice if using a macro requires users to add other dependencies :)

Reply:
Yep, this was just a quick hack for testing before trying to re-export it correctly